CMakeLists.txt

```diff
@@ -159,12 +159,12 @@ endif()
 
 set(NOVELRT_LIBRARY_HEADERS
   ${NOVELRT_HEADERS}
-  include/NovelRT/NovelRT-C/NovelRunner_C.h
+  #include/NovelRT/NovelRT-C/NovelRunner_C.h
```

Comment:
We should either remove this (and redo it in the future when we cross that road) or move it to a `libNovelRT-C` project